packages/rancher-gatekeeper/generated-changes/patch/values.yaml.patch

```diff
@@ -1,44 +1,41 @@
 --- charts-original/values.yaml
 +++ charts/values.yaml
-@@ -37,10 +37,13 @@
+@@ -39,10 +39,10 @@
  admissionEventsInvolvedNamespace: false
  auditEventsInvolvedNamespace: false
  resourceQuota: true
 -image:
 -  repository: openpolicyagent/gatekeeper
 -  crdRepository: openpolicyagent/gatekeeper-crds
--  release: v3.12.0
+-  release: v3.13.0
 +images:
 +  gatekeeper:
 +    repository: rancher/mirrored-openpolicyagent-gatekeeper
-+    tag: v3.12.0
-+  gatekeepercrd:
-+    repository: rancher/mirrored-openpolicyagent-gatekeeper-crds
-+    tag: v3.12.0
++    tag: v3.13.0
```

Comment:
Did we remove rancher/mirrored-openpolicyagent-gatekeeper-crds image from values.yaml ? Was this intential ?